src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
That could work, but you can't avoid reliance. Currently both dosemu and the install script need the detection result. Dosemu needs it to install either the proprietary or the freedos variants of autoexec.bat+config.sys and the install script to call the right installation routine.

Installing the correct variant of those two with the install script would avoid the need for dosemu to be aware of this result.